README.md

```diff
@@ -593,6 +612,53 @@ All components can be imported and used independently for advanced use cases. Th
 
 Direct interface to the Payments contract for token operations and operator approvals.
 
+#### Understanding Payment Rails
+
+Payment rails are continuous payment streams between clients and service providers that are created automatically when data sets are established. Each data set has associated payment rails (one for PDP storage, optionally additional ones for CDN services).
+
+**How Rails Work vs Traditional Payment:**
+
+Unlike traditional upfront payment models, rails use a **lockup mechanism** that acts as a guarantee without being a separate payment pool:
+
+1. **The Lockup Guarantee**: When a rail is created, the system calculates a lockup requirement:
+   - Formula: `lockup = paymentRate × lockupPeriod` (e.g., 10 days of payments)
+   - Example: Storing 1 GiB at ~0.0000565 USDFC/epoch (based on 5 USDFC/TiB/month) requires ~1.63 USDFC lockup
+   - This amount acts as a **minimum balance floor** - you can't withdraw below it
+
+2. **How Payments Actually Work**:
+   - **During Normal Operation**: Settlements draw from your general funds, NOT from the lockup
```

Comment:
This reads as if the lockup was never "withdrawn" from / used, is that correct? Or is there a section missing about "During Not Normal Operation"?